### PR TITLE
Ci/fix new ci

### PR DESCRIPTION
**Pull request by marcvs**
Make things work with the new ci

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -34,6 +34,9 @@ AGENT_SERVICE = oidc-agent-service
 PROMPT        = oidc-prompt
 
 VERSION   ?= $(shell cat VERSION)
+TILDE_VERSION := $(shell echo $(VERSION) | sed s/-pr/~pr/)
+BASE_VERSION := $(shell head debian/changelog  -n 1 | cut -d \( -f 2 | cut -d \) -f 1 | cut -d \- -f 1)
+DEBIAN_VERSION := $(shell head debian/changelog  -n 1 | cut -d \( -f 2 | cut -d \) -f 1 | sed s/-[0-9][0-9]*//)
```
Comment (zachmann) [inline Makefile:39]:
```suggestion
```

Those are not used apart from in the info debugging.

File: Makefile [hunk 2]
```diff
@@ -947,6 +950,14 @@ cleanapi:
...
 
 # Packaging
+info:
+	@echo "DESTDIR:         $(DESTDIR)"
+	@echo "INSTALLDIRS:     $(INSTALLDIRS)"
+	@echo "VERSION:         $(VERSION)"
+	@echo "TILDE_VERSION:   $(TILDE_VERSION)"
+	@echo "RPM_VERSION:     $(RPM_VERSION)"
+	@echo "DEBIAN_VERSION:  $(DEBIAN_VERSION)"
+	@echo "BASE_VERSION:    ${BASE_VERSION}"
```
Comment (zachmann) [inline Makefile:960]:
```suggestion
```
Again not needed

File: .gitlab-ci.yml [hunk 3]
```diff
@@ -2,11 +2,28 @@
...
+  # REPO_USER
+  # REPO_HOST
+  # PUBLISH_BUILD_RESULTS_REPO: 'devel'
+  # PUBLISH_TESTED_RESULTS_REPO: 'prerel'
+  # PUBLISH_MASTER_RESULTS_REPO: 'for-prod'
+  # PREREL_BRANCH_NAME: 'prerel'
+  # See generic-ci.yml -> .build-definition for the full list
+  # DOCKER_IMAGE_VERSION
+  # DOCKER_IMAGE_VERSION_WINDOWS
+  PREREL_BRANCH_NAME: 'ci/fix-new-ci'
```
Comment (zachmann) [inline .gitlab-ci.yml:26]: I guess this should be changed to `prerel` or commented out, depending on how things work

**Review by zachmann — CHANGES_REQUESTED** [review]

File: Makefile (hunk 2 above)
Comment (marcvs) [reply Makefile:960]: Needed in included docker/debian.mk

File: .gitlab-ci.yml (hunk 3 above)
Comment (marcvs) [reply .gitlab-ci.yml:26]: Done

**Pull request merged**